packages/vscode-extension/src/debugging/DebugSession.ts

```diff
@@ -89,16 +91,21 @@ export class DebugSession implements Disposable {
     return this.currentWsTarget;
   }
 
+  private startParentSessionPromise: Promise<vscode.DebugSession> | undefined;
+
   public async startParentDebugSession() {
     assert(
       !this.jsDebugSession,
       "Cannot start parent debug session when js debug session is already running"
     );
-    const newParentDebugSession = await startDebugging(
+    if (this.startParentSessionPromise) {
+      this.startParentSessionPromise.then((session) => debug.stopDebugging(session));
+    }
```

Comment:
wouldn't it make more sense to add `cancelToken` as a parameter to startDebugging?

Reply:
Possibly, I haven't thought of that. I'll look into it.